### PR TITLE
Nginxingress

### PR DESCRIPTION
**Pull request by duallain**

### CODE REVIEW

File: mozmeao-fr/bedrock-prod/ingress.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      http:
+        paths:
+          - backend:
+              serviceName: bedrock-prod-ingress
+              servicePort: 8000
+            path: /
+  # This section is only required if TLS is to be enabled for the Ingress
+  tls:
+      - hosts:
+          - "bedrock.prod.fr.moz.works"
```
Comment (pmac) [inline mozmeao-fr/bedrock-prod/ingress.yml:55]: does this also need to mention `www.mozilla.org`?

Comment (duallain) [reply mozmeao-fr/bedrock-prod/ingress.yml:62]: I don't believe so.  Cloudflare terminates SSL and then makes requests to the origin dns address, bedrock-prod.moz.works. `bedrock-prod.frankfurt.moz.works` currently only has certs that are valid for moz.works.

**Review by limed — APPROVED** [review]

File: mozmeao-fr/bedrock-prod/ingress.yml (hunk 1 above)
Comment (pmac) [reply mozmeao-fr/bedrock-prod/ingress.yml:62]: Ah, yeah, for TLS, right. We would need to configure the WMO domain for ingress routing though I'm guessing?

Comment (duallain) [reply mozmeao-fr/bedrock-prod/ingress.yml:62]: @pmac I don't believe so I think cloudflare is directly accessing the origin, but it also doesn't costs me anything (no perf hits, not really that much extra work, etc) so I'm going to err on the side of caution and push a PR that includes it.  We can always pull that later if the logs don't show that host being used.

**Pull request merged**